test/with_dummyserver/test_connectionpool.py

```diff
@@ -372,7 +392,7 @@ def test_bad_connect(self):
             pool.request('GET', '/', retries=5)
             self.fail("should raise timeout exception here")
         except MaxRetryError as e:
-            self.assertTrue(isinstance(e.reason, ProtocolError), e.reason)
+            self.assertEqual(type(e.reason), NewConnectionError)
```

Comment:
Yes, not a terrible idea, but I strongly suspect it'll devolve into a big several-day ordeal of cleanup that really needs to happen anyways. The structure of the tests is a hot mess right now.